src/KubeOps/Operator/Kubernetes/ResourceWatcher{TEntity}.cs

```diff
@@ -25,6 +25,8 @@ internal class ResourceWatcher<TEntity> : IDisposable, IResourceWatcher<TEntity>
     private CancellationTokenSource? _cancellation;
     private Watcher<TEntity>? _watcher;
 
+    private const int MaxRetriesAttempts = 39;
```

Comment:
It is not random I calculated the max number of retries. From 39 up it will start to throw overflow exceptions.

Reply:
It fine for me, I just thought because of the random component in the exponential backoff it may throw if certain conditions happen. That's why I thought to take 20 or 30. Just to be on the safe side.